api/libproject/devstart.rb

```diff
@@ -10,21 +11,95 @@ def dev_up()
   common.run_inline_swallowing_interrupt %W{docker-compose logs -f api}
 end
 
-def rebuild_image()
+def rebuild_image(args)
   common = Common.new
   common.docker.requires_docker
 
   common.run_inline %W{docker-compose build}
 end
 
+def auth_login(account)
```

Comment:
Consider passing `--account` and `--project` to the relevant gcloud commands downstream, to avoid making changes to default gcloud settings (or consider saving and restoring).

(Now, I'm not sure how you can avoid a `gcloud auth application-default login` to make `gsutil` work — but in your current approach, I'm also not sure how `gsutil` is getting credentials to perform the  `cp`... https://cloud.google.com/storage/docs/authentication suggests you need `gcloud auth application-default login` to provide user credentials, but I don't see that here)